packages/zipkin/src/InetAddress.js

```diff
@@ -21,6 +19,8 @@ class InetAddress {
 }
 
 InetAddress.getLocalAddress = function getLocalAddress() {
+  // eslint-disable-line global-require
+  const networkAddress = require('network-address');
```

Comment:
It would fail in the current implementation. It is only slightly better, but I honestly don't know of any solution for getting the network address in such a setting.